anagrafica/modelli.py

```diff
@@ -34,17 +34,20 @@ class Persona(ModelloCancellabile, ConGeolocalizzazioneRaggio):
     )
 
     # Stati
+    # Valutiamo se mettere militare e IV tra gli stati o Appartenenze
```

Comment:
Mi sa che andranno solo in appartenenze, in quanto una persona potrebbe avere piu' ruoli.